packages/rocketchat-api/server/v1/settings.js

```diff
@@ -1,23 +1,23 @@
 import _ from 'underscore';
 
 // settings endpoints
-RocketChat.API.v1.addRoute('settings.public', { authRequired: false }, {
+RocketChat.API.v1.addRoute('settings.public', {authRequired: false}, {
```

Comment:
Can we add this white space back in all along the file? That way the diff is less and only shows what really changed, makes it difficult to review and tell what changed in git history and git blames.